jobs/webcompat-kb/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.8
+MAINTAINER <kberezina@mozilla.com>
+
+# https://github.com/mozilla-services/Dockerflow/blob/master/docs/building-container.md
+ARG USER_ID="10001"
+ARG GROUP_ID="app"
+ARG HOME="/app"
+
+ENV HOME=${HOME}
+RUN groupadd --gid ${USER_ID} ${GROUP_ID} && \
+    useradd --create-home --uid ${USER_ID} --gid ${GROUP_ID} --home-dir ${HOME} ${GROUP_ID}
+
+WORKDIR ${HOME}
```

Comment:
I haven't modified this file since it was generated by `create_job` script based on a template that every job in this repository follows, so it's probably fine to leave it as is.